openfisca_aotearoa/tests/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.7.2-stretch
+WORKDIR /app
+
+COPY setup.py /app/setup.py
+COPY setup.cfg /app/setup.cfg
+COPY requirements.txt /app/requirements.txt
+
+RUN pip install --upgrade pip && \
+  pip install numpy'<1.16,>=1.11' && \
```

Comment:
without this line, a much more recent numpy is attempted to be installed using easy-install, and complains about not working with this python version